Module4_Labs/Lab3_File_System/Cards/31.md

```diff
@@ -3,6 +3,7 @@
 <!--badges={Python:11,Algorithms:15}-->
 
 <!--concepts={directedGraphs, introToGraphs, useOfGraphs}-->
+**Step 1: Use the hieracrhy_pos() function to order the positioning of each node**
```

Comment:
again do steps by card, the number of steps should reset with each card

Reply:
I resolved this issue.